k8s/service_controller/utils/utils.go

```diff
@@ -51,3 +53,11 @@ func ResponseBodyToObject(r *http.Response, object interface{}) error {
 func ExtractVarFromRequest(r *http.Request, varName string) string {
 	return mux.Vars(r)[varName]
 }
+
+func KubeCreateResource(r io.Reader) (string, error) {
```

Comment:
Do me a favor... add a comment here noting that while we're using `kubectl` today, this is a temporary thing and we're going to switch to normal REST calls at some point.  Just so people know what our plans are.